sdk/spring/azure-spring-cloud-stream-binder-eventhubs/README.md

```diff
@@ -63,59 +65,78 @@ Name | Description | Required | Default
 
  #### Event Hub Producer Properties ####
 
- It supports the following configurations with the format of `spring.cloud.stream.eventhub.bindings.<channelName>.producer`.
- 
- **_sync_**
+It supports the following configurations with the format of `spring.cloud.stream.eventhub.bindings.<channelName>.producer`.
  
- Whether the producer should act in a synchronous manner with respect to writing messages into a stream. If true, the 
- producer will wait for a response from Event Hub after a send operation.
+**_sync_**
 
- Default: `false`
+Whether the producer should act in a synchronous manner with respect to writing messages into a stream. If true, the 
+producer will wait for a response from Event Hub after a send operation.
 
- **_send-timeout_**
+Default: `false`
 
- Effective only if `sync` is set to true. The amount of time to wait for a response from Event Hub after a send operation, in milliseconds.
+**_send-timeout_**
 
- Default: `10000`
- 
- #### Event Hub Consumer Properties ####
+Effective only if `sync` is set to true. The amount of time to wait for a response from Event Hub after a send operation, in milliseconds.
+
+Default: `10000`
+
+**_partition-count_**
```

Comment:
is this a producer property? I mean, is this property `spring.cloud.stream.eventhub.bindings.<channelName>.producer.partition-coung`?

Reply:
To be precise, it should be `spring.cloud.stream.bindings.<channelName>.producer.partition-count`.